doc/changelog.rst

```diff
@@ -37,6 +37,21 @@ MJX
 - Added support for spatial tendons with internal sphere and cylinder wrapping.
 - Fix a bug with box-box collisions :github:issue:`2356`.
 
+Python bindings
+^^^^^^^^^^^^^^^
+
+ .. youtube:: 
```

Comment:
I will add a video once everything is finalized.

Reply:
let's move the changes in this file to a child PR?